avtozip/store/templates/store/index.html

```diff
@@ -2,7 +2,36 @@
 
 {% block title %}AvtoZip Store{% endblock %}
 
+{% block staticdata %}
+  {% load staticfiles %}
```

Comment:
Move `load staticfiles` to the very top (see django examples where load should be placed)
use `{{ super }}` directive here to call prepend the block from parent
rename block to `css`